File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Grid;

namespace WindowsApplication1
{
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1()
        {
            InitializeComponent();


            WindowState = FormWindowState.Maximized;
            gridView1.OptionsView.ColumnAutoWidth = false;
            gridView1.OptionsBehavior.EditorShowMode = DevExpress.Utils.EditorShowMode.MouseDownFocused;
            gridView1.OptionsSelection.MultiSelect = true;
            gridView1.OptionsSelection.MultiSelectMode = GridMultiSelectMode.CellSelect;
            new DataSourceHelper(gridView1, 50, 50);
            SelectedCellsBorderHelper selectedCellsHelper = new SelectedCellsBorderHelper(gridView1);
            new DragCellsValuesHelper(selectedCellsHelper);
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Views.Grid

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			gridView1.OptionsView.ColumnAutoWidth = False
			WindowState = FormWindowState.Maximized
			gridView1.OptionsBehavior.EditorShowMode = DevExpress.Utils.EditorShowMode.MouseDownFocused
			gridView1.OptionsSelection.MultiSelect = True
			gridView1.OptionsSelection.MultiSelectMode = GridMultiSelectMode.CellSelect
			Dim TempDataSourceHelper As DataSourceHelper = New DataSourceHelper(gridView1, 50, 50)
			Dim selectedCellsHelper As New SelectedCellsBorderHelper(gridView1)
			Dim TempDragCellsValuesHelper As DragCellsValuesHelper = New DragCellsValuesHelper(selectedCellsHelper)

		End Sub
	End Class
End Namespace